src/commands/Starboard/star.ts

```diff
@@ -31,13 +29,13 @@ const MEDALS = ['🥇', '🥈', '🥉'];
 	]
 ])
 export default class extends SkyraCommand {
-	public async random(message: KlasaMessage, [member]: [GuildMember?]): Promise<KlasaMessage | KlasaMessage[]> {
-		const minimum = message.guild!.settings.get(GuildSettings.Starboard.Minimum);
+	public async random(message: GuildMessage, [member]: [GuildMember?]): Promise<Message | Message[]> {
+		const minimum = await message.guild.readSettings(GuildSettings.Starboard.Minimum);
```

Comment:
Return language, adapt negatives with throw language.get for consistency with other files